src/app/App.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { SyntheticEvent, useEffect, useState } from 'react';
+import { useAppDispatch, useAppSelector } from 'hooks/useRedux';
+import UsersAsync from 'store/users/usersAsync';
+import User from 'models/User';
+import Post from 'models/Post';
+import { selectUsers } from 'store/users/usersSelectors';
+import { selectSelectedPost } from 'store/posts/postsSelectors';
+import { PostsList } from 'components/PostsList';
+import { PostDetails } from 'components/PostDetails';
+import { Autocomplete, TextField } from '@mui/material';
+
+export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+
+  const users: User[] | null = useAppSelector(selectUsers);
+  const selectedPost: Post | null = useAppSelector(selectSelectedPost);
```

Comment:
it's already typified in redux, no need to do it here again

Reply:
@vlad-demchuk Do you think it's a big deal to disapprove a task?